standalone-metastore/metastore-client/src/main/java/org/apache/hadoop/hive/metastore/client/HookEnabledMetaStoreClient.java

```diff
@@ -325,6 +327,21 @@ public List<Partition> dropPartitions(String catName, String dbName, String tblN
     return delegate.dropPartitions(catName, dbName, tblName, partExprs, options, context);
   }
 
+  @Override
```

Comment:
you've dropped the old signature in `SessionHiveMetaStoreClient`, but not here, why?
````
public List<Partition> dropPartitions(String catName, String dbName, String tblName,
      List<Pair<Integer, byte[]>> partExprs, PartitionDropOptions options, EnvironmentContext context)
````